README.md

```diff
@@ -214,6 +214,33 @@ As administrator
 pip install --user robotframework-autoitlibrary
 ```
 
+General usage
+====
+
+The framework comes with its own CLI utility exposed through the make command. To enlist all commands available run:
+
+```
+make
+```
+
+Settings / Config
+====
+
+The framework has 3 files that serve its settings.
+
+1. settings.robot (Default distributed settings that ship as standard).
+2. settings-override.py (A templated custom override file that is not checked in, and can be modified).
+3. secrets.robot (A templated file that is not checked in, and holds all sensitive information such as passwords).
+
+*Note: Any variable throughout the framework can be overriden from the override-settings file not just the settings necessarily.*
+
+To create/edit the templated files, use the provided make commands:
+
+```
+make (edit-)override-settings
```

Comment:
I think having the `(edit-)` part is potentially confusing as could be interpreted that `make (edit-)override-settings` needs to be literally typed in. Might be better to explicitly list the options individually.

Reply:
Done